app/src/views/Landing/Landing.tsx

```diff
@@ -1,9 +1,132 @@
-import React from 'react'
+import React from 'react';
+import { Button } from '@chakra-ui/button';
+import { Image } from '@chakra-ui/react'
+import { useColorModeValue } from '@chakra-ui/color-mode';
+import { Box, Container, Flex, Heading, HStack, List, ListIcon, ListItem, Stack, Text, VStack } from '@chakra-ui/layout';
+import { useBreakpointValue } from '@chakra-ui/media-query';
+import { CheckIcon } from '@chakra-ui/icons';
+import { Link } from 'react-router-dom';
+import { routes } from '../../routes/routes';
+import SecureImage from '../../assets/SecureImage.svg';
+import DoctorsInComputerImage from '../../assets/DoctorsInComputerImage.svg';
+import DoctorWithPhoneImage from '../../assets/DoctorWithPhoneImage.svg';
+import PersonWithComputerImage from '../../assets/PersonWithComputerImage.svg';
+
+const bulletList = [{
+  title: 'Con una cuenta de paciente:',
+  listItems: [
+    'Administrar tu historia médica',
+    'Compartir tu historia médica con tu doctor',
+    'Organizar tus estudios médicos'
+  ]
+},
+{
+  title: 'Con una cuenta de doctor:',
+  listItems: [
+    'Ver las historias médicas de tus pacientes',
+    'Organizar las notas de tus pacientes',
+    'Administrar tu historia médica',
+    'Organizar tus estudios médicos'
+  ]
+}, 
+]
 
 export const Landing = () => {
-    return (
-        <div>
-            <h1>Landing page!!!!</h1>
-        </div>
-    )
+
+const landingImage = useBreakpointValue({ base: DoctorWithPhoneImage, md: DoctorsInComputerImage })
+
+  return (
+    <Container
+      w="100vw"
+      h="100%"
+      maxW="95%"
+      d={'flex'}
+      flexGrow={1}
+      mx={{base:'1em', md:'2em'}}
+    >
+      <Flex
+        flexGrow={1}
+        align={'flex-start'}
+        justify={'center'}
+        py={12}
+        w={'100%'}
+      >
+        <VStack align="stretch" spacing={{base: '5em', md:'10em'}} w={'100%'}>
+          {/**************  1st Section **************/}
```

Comment:
Cambiaría este tipo de comentarios por frases con mejor significado. Por ejemplo, "Call to action section", "Safety features section", etc.